spec/spec_helper.rb

```diff
@@ -1,3 +1,13 @@
+require 'coveralls'
+require 'simplecov'
```

Comment:
this should be after `bundler/setup` because otherwise it can load different version than in the bundle (when executed without bundle exec).